src/LM_fmt.c

```diff
@@ -71,7 +71,7 @@ static void init(struct fmt_main *self)
 
 static char *prepare(char *fields[10], struct fmt_main *self)
 {
-	if (fields[2] && strlen(fields[2]) == 32)
+	if (fields[1][0] != '$' && fields[2] && strlen(fields[2]) == 32)
```

Comment:
Maybe we could use `(fields[1][0] != '$' || !strchr(&field[1][1], '$'))` - that is, exclude only lines that have at least two dollar signs in that field.

Reply:
I'm concerned about performance issues in prepare functions (which is a bit of a problem already). Maybe I worry too much.